OpenSim/Simulation/Wrap/WrapCylinder.cpp

```diff
@@ -118,13 +118,11 @@ void WrapCylinder::extendFinalizeFromProperties()
     Super::extendFinalizeFromProperties();
 
     // maybe set a parent pointer, _body = aBody;
-    if (get_radius() < 0.0)
-    {
-        string errorMessage = getProperty_radius().getName()
-            + " is not specified or is negative.";
-        OPENSIM_THROW_FRMOBJ(InvalidPropertyValue,
-            getProperty_radius().getName(), errorMessage);
-    }
+    OPENSIM_THROW_IF_FRMOBJ(
+        get_radius() < 0,
+        InvalidPropertyValue,
+        getProperty_radius().getName(),
+        "Radius must be specified and cannot be less than zero");
```

Comment:
Just curious: What led you to update these error checks? It doesn't look like you changed the constructor to `InvalidPropertyValue` or anything like that.

Reply:
I appreciate the additions/correction (goes beyond the call of duty for this PR) as part of the sweep to add meaningful messages. I think it is fine leave it in here- please don't spend the time to separate at this point.

Reply:
Of course; I did not mean that these changes should be separated. Ignore the question.

Reply:
Ajay wanted the exceptions in the `MuscleFixedWidthPennationModel` and `MuscleRebellionActivationDynamicModel` `extendFinalizeFromProperties()` methods changed to `InvalidPropertyValue`, so I changed them by following examples elsewhere in the repo. I then got more comments saying that my use of `InvalidPropertyValue` was poor, so I updated all uses of `InvalidPropertyValue` in the repo so that doesn't happen again.

Reply:
Thanks @tkuchida!